x-pack/plugins/enterprise_search/public/applications/app_search/components/search_ui/types.ts

```diff
@@ -6,9 +6,9 @@
  */
 
 export enum ActiveField {
-  Title,
-  Filter,
-  Sort,
-  Url,
-  None,
+  Title = 'title',
+  Filter = 'filter',
+  Sort = 'sort',
+  Url = 'url',
+  None = '',
```

Comment:
Just checking, was this meant to go in the SVG commit? Not sure if it significantly impacts the PR so no worries either way

Reply:
Yeah, these get used to create the CSS class names for the SVG, so I needed actual string values.

Reply:
Ahhh gotcha! Thanks for explaining 🤓